cpp/src/arrow/util/trie-benchmark.cc

```diff
@@ -191,31 +191,28 @@ static void BenchmarkInlinedTrieLookups(
   state.SetItemsProcessed(state.iterations() * lookups.size());
 }
 
-static void BM_TrieLookupFound(benchmark::State& state) {  // NOLINT non-const reference
+static void TrieLookupFound(benchmark::State& state) {  // NOLINT non-const reference
   BenchmarkTrieLookups(state, {"N/A", "null", "-1.#IND", "N/A"});
 }
 
-static void BM_TrieLookupNotFound(
-    benchmark::State& state) {  // NOLINT non-const reference
+static void TrieLookupNotFound(benchmark::State& state) {  // NOLINT non-const reference
   BenchmarkTrieLookups(state, {"None", "1.0", "", "abc"});
 }
 
-static void BM_InlinedTrieLookupFound(
+static void InlinedTrieLookupFound(
     benchmark::State& state) {  // NOLINT non-const reference
   BenchmarkInlinedTrieLookups(state, {"N/A", "null", "-1.#IND", "N/A"});
 }
 
-static void BM_InlinedTrieLookupNotFound(
+static void InlinedTrieLookupNotFound(
     benchmark::State& state) {  // NOLINT non-const reference
   BenchmarkInlinedTrieLookups(state, {"None", "1.0", "", "abc"});
 }
 
-static const int kRepetitions = 2;
-
-BENCHMARK(BM_TrieLookupFound)->Repetitions(kRepetitions);
-BENCHMARK(BM_TrieLookupNotFound)->Repetitions(kRepetitions);
-BENCHMARK(BM_InlinedTrieLookupFound)->Repetitions(kRepetitions);
-BENCHMARK(BM_InlinedTrieLookupNotFound)->Repetitions(kRepetitions);
+BENCHMARK(TrieLookupFound);
+BENCHMARK(TrieLookupNotFound);
+BENCHMARK(InlinedTrieLookupFound);
+BENCHMARK(InlinedTrieLookupNotFound);
```

Comment:
The "inlined trie" benchmarks are reference benchmarks (to compare against a hand-written specialized version of trie lookup).